### PR TITLE
[QUEUES] Fixing formatting

### PR DESCRIPTION
**Pull request by Maddy-Cloudflare**
PCX-7721

### CODE REVIEW

File: content/queues/platform/configuration.md [hunk 1]
```diff
@@ -32,11 +32,11 @@ To enable Producer Worker to Queue communication, set up a binding in your `wran
 
 {{<definitions>}}
 
-- {{<code>}}queue{{<param-type>}}string{{</param-type>}}{{</code>}}
+- {{<code>}}queue{{</code>}}{{<param-type>}}string{{</param-type>}}
```
Comment (pedrosousa) [inline content/queues/platform/configuration.md]:
```suggestion
- {{<code>}}queue{{</code>}} {{<type>}}string{{</type>}}
```

Actually, `param-type` should be used inside a `code` context (the CSS styles expect this to happen). It is commonly used for parameter data types when stating the full signature of a function, like in this example:
https://github.com/cloudflare/cloudflare-docs/blob/production/content/queues/platform/javascript-apis.md?plain=1#L69
image

When creating a reference entry for a single property/parameter/config entry, like here, and assuming that you wish to keep a similar formatting (visually) between property data types and function return types, you can use `type` for both. `type` can be used outside a `code` context.

Currently, we're getting different styles (regular font vs monospaced) when using `param-type` and `type`, respectively:
image

The change I'm proposing would affect most of the changes in this PR.

Comment (Maddy-Cloudflare) [reply content/queues/platform/configuration.md]: Thanks for explaining this, @pedrosousa 😄

File: content/queues/platform/configuration.md [hunk 2]
```diff
@@ -32,11 +32,11 @@ To enable Producer Worker to Queue communication, set up a binding in your `wran
 
 {{<definitions>}}
 
-- {{<code>}}queue{{<param-type>}}string{{</param-type>}}{{</code>}}
+- {{<code>}}queue{{</code>}} {{<type>}}string{{</type>}}
 
   - The name of the Queue.
 
-- {{<code>}}binding{{<param-type>}}string{{</param-type>}}{{</code>}}
+- {{<code>}}binding{{</code>}}{{<param-type>}}string{{</param-type>}}
```
Comment (pedrosousa) [inline content/queues/platform/configuration.md]:
```suggestion
- {{<code>}}binding{{</code>}} {{<type>}}string{{</type>}}
```

File: content/queues/platform/configuration.md [hunk 3]
```diff
@@ -59,29 +59,29 @@ Refer to [Limits](/queues/platform/limits) to review the maximum values for each
 
 {{<definitions>}}
 
-- {{<code>}}queue{{<param-type>}}string{{</param-type>}}{{</code>}}
+- {{<code>}}queue{{</code>}}{{<param-type>}}string{{</param-type>}}
```
Comment (pedrosousa) [inline content/queues/platform/configuration.md]:
```suggestion
- {{<code>}}queue{{</code>}} {{<type>}}string{{</type>}}
```

File: content/queues/platform/configuration.md [hunk 4]
```diff
@@ -59,29 +59,29 @@ Refer to [Limits](/queues/platform/limits) to review the maximum values for each
...
 
   - The name of the Queue.
 
-- {{<code>}}max_batch_size{{<param-type>}}number{{</param-type>}}{{</code>}} {{<prop-meta>}}optional{{</prop-meta>}}
+- {{<code>}}max_batch_size{{</code>}}{{<param-type>}}number{{</param-type>}} {{<prop-meta>}}optional{{</prop-meta>}}
```
Comment (pedrosousa) [inline content/queues/platform/configuration.md]:
```suggestion
- {{<code>}}max_batch_size{{</code>}} {{<type>}}number{{</type>}} {{<prop-meta>}}optional{{</prop-meta>}}
```

File: content/queues/platform/configuration.md [hunk 5]
```diff
@@ -59,29 +59,29 @@ Refer to [Limits](/queues/platform/limits) to review the maximum values for each
...
 
   - The maximum number of messages allowed in each batch.
 
-- {{<code>}}max_batch_timeout{{<param-type>}}number{{</param-type>}}{{</code>}} {{<prop-meta>}}optional{{</prop-meta>}}
+- {{<code>}}max_batch_timeout{{</code>}}{{<param-type>}}number{{</param-type>}} {{<prop-meta>}}optional{{</prop-meta>}}
```
Comment (pedrosousa) [inline content/queues/platform/configuration.md]:
```suggestion
- {{<code>}}max_batch_timeout{{</code>}} {{<type>}}number{{</type>}} {{<prop-meta>}}optional{{</prop-meta>}}
```

File: content/queues/platform/configuration.md [hunk 6]
```diff
@@ -59,29 +59,29 @@ Refer to [Limits](/queues/platform/limits) to review the maximum values for each
...
 
   - The maximum number of seconds to wait until a batch is full.
 
-- {{<code>}}max_retries{{<param-type>}}number{{</param-type>}}{{</code>}} {{<prop-meta>}}optional{{</prop-meta>}}
+- {{<code>}}max_retries{{</code>}}{{<param-type>}}number{{</param-type>}} {{<prop-meta>}}optional{{</prop-meta>}}
```
Comment (pedrosousa) [inline content/queues/platform/configuration.md]:
```suggestion
- {{<code>}}max_retries{{</code>}} {{<type>}}number{{</type>}} {{<prop-meta>}}optional{{</prop-meta>}}
```

File: content/queues/platform/configuration.md [hunk 7]
```diff
@@ -59,29 +59,29 @@ Refer to [Limits](/queues/platform/limits) to review the maximum values for each
...
 
   - The maximum number of retries for a message, if it fails or [`retryAll()`](/queues/platform/javascript-apis/#messagebatch) is invoked.
 
-- {{<code>}}dead_letter_queue{{<param-type>}}string{{</param-type>}}{{</code>}} {{<prop-meta>}}optional{{</prop-meta>}}
+- {{<code>}}dead_letter_queue{{</code>}}{{<param-type>}}string{{</param-type>}} {{<prop-meta>}}optional{{</prop-meta>}}
```
Comment (pedrosousa) [inline content/queues/platform/configuration.md]:
```suggestion
- {{<code>}}dead_letter_queue{{</code>}} {{<type>}}string{{</type>}} {{<prop-meta>}}optional{{</prop-meta>}}
```

File: content/queues/platform/configuration.md [hunk 8]
```diff
@@ -59,29 +59,29 @@ Refer to [Limits](/queues/platform/limits) to review the maximum values for each
...
 
   - The name of another Queue to send a message if it fails processing at least `max_retries` times.
   - If a `dead_letter_queue` is not defined, messages that repeatedly fail processing will eventually be discarded.
   - If there is no Queue with the specified name, it will be created automatically.
 
-- {{<code>}}max_concurrency{{<param-type>}}number{{</param-type>}}{{</code>}} {{<prop-meta>}}optional{{</prop-meta>}}
+- {{<code>}}max_concurrency{{</code>}}{{<param-type>}}number{{</param-type>}} {{<prop-meta>}}optional{{</prop-meta>}}
```
Comment (pedrosousa) [inline content/queues/platform/configuration.md]:
```suggestion
- {{<code>}}max_concurrency{{</code>}} {{<type>}}number{{</type>}} {{<prop-meta>}}optional{{</prop-meta>}}
```

File: content/queues/platform/javascript-apis.md [hunk 9]
```diff
@@ -85,7 +85,7 @@ type MessageSendRequest<Body = any> = {
 
 {{<definitions>}}
 
-- {{<code>}}body{{<param-type>}}any{{</param-type>}}{{</code>}}
+- {{<code>}}body{{</code>}}{{<param-type>}}any{{</param-type>}}
```
Comment (pedrosousa) [inline content/queues/platform/javascript-apis.md]:
```suggestion
- {{<code>}}body{{</code>}} {{<type>}}any{{</type>}}
```

File: content/queues/platform/javascript-apis.md [hunk 10]
```diff
@@ -147,19 +147,19 @@ interface MessageBatch<Body = any> {
 
 {{<definitions>}}
 
-- {{<code>}}queue{{<param-type>}}string{{</param-type>}}{{</code>}}
+- {{<code>}}queue{{</code>}}{{<param-type>}}string{{</param-type>}}
```
Comment (pedrosousa) [inline content/queues/platform/javascript-apis.md]:
```suggestion
- {{<code>}}queue{{</code>}} {{<type>}}string{{</type>}}
```

File: content/queues/platform/javascript-apis.md [hunk 11]
```diff
@@ -147,19 +147,19 @@ interface MessageBatch<Body = any> {
...
 
   - The name of the Queue that belongs to this batch.
 
-- {{<code>}}messages{{<param-type>}}Message[]{{</param-type>}}{{</code>}}
+- {{<code>}}messages{{</code>}}{{<param-type>}}Message[]{{</param-type>}}
```
Comment (pedrosousa) [inline content/queues/platform/javascript-apis.md]:
```suggestion
- {{<code>}}messages{{</code>}} {{<type>}}Message[]{{</type>}}
```

File: content/queues/platform/javascript-apis.md [hunk 12]
```diff
@@ -181,24 +181,24 @@ interface Message<Body = any> {
 
 {{<definitions>}}
 
-- {{<code>}}id{{<param-type>}}string{{</param-type>}}{{</code>}}
+- {{<code>}}id{{</code>}}{{<param-type>}}string{{</param-type>}}
```
Comment (pedrosousa) [inline content/queues/platform/javascript-apis.md]:
```suggestion
- {{<code>}}id{{</code>}} {{<type>}}string{{</type>}}
```

File: content/queues/platform/javascript-apis.md [hunk 13]
```diff
@@ -181,24 +181,24 @@ interface Message<Body = any> {
...
 
   - A unique, system-generated ID for the message.
 
-- {{<code>}}timestamp{{<param-type>}}Date{{</param-type>}}{{</code>}}
+- {{<code>}}timestamp{{</code>}}{{<param-type>}}Date{{</param-type>}}
```
Comment (pedrosousa) [inline content/queues/platform/javascript-apis.md]:
```suggestion
- {{<code>}}timestamp{{</code>}} {{<type>}}Date{{</type>}}
```

File: content/queues/platform/javascript-apis.md [hunk 14]
```diff
@@ -181,24 +181,24 @@ interface Message<Body = any> {
...
 
   - A timestamp when the message was sent.
 
-- {{<code>}}body{{<param-type>}}any{{</param-type>}}{{</code>}}
+- {{<code>}}body{{</code>}}{{<param-type>}}any{{</param-type>}}
```
Comment (pedrosousa) [inline content/queues/platform/javascript-apis.md]:
```suggestion
- {{<code>}}body{{</code>}} {{<type>}}any{{</type>}}
```

**Review by pedrosousa — APPROVED** [review]

**Pull request merged**